tests/FakerDotNet.Tests/Fakers/SpaceFakerTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿using FakeItEasy;
+using FakerDotNet.Data;
+using FakerDotNet.Fakers;
+using FakerDotNet.Tests.Helpers;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace FakerDotNet.Tests.Fakers
+{
+    [TestFixture]
+    [Parallelizable]
+    public class SpaceFakerTests
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            _fakerContainer = A.Fake<IFakerContainer>();
+            _spaceFaker = new SpaceFaker(_fakerContainer);
+        }
+
+        private static readonly SpaceData Data = new SpaceData();
+
+        private IFakerContainer _fakerContainer;
+        private ISpaceFaker _spaceFaker;
+
+        [Test]
+        public void Planet_returns_a_planet()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(Data.Planets)))
+                .Returns(Data.Planets.First());
+
+            Assert.AreEqual(Data.Planets.First(), _spaceFaker.Planet());
```

Comment:
The pattern that I have been following is to return the value that is in the docs and do the assertion on that. So here it would be `Assert.AreEqual("Venus", _spaceFaker.Planet());`